From 47ca84c4b3a7e68d45586be06a8cae43ac34141d Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:02 -0400
Subject: [redhat] kabi: Adding symbol scsi_is_fc_rport (drivers/scsi/scsi_transport_fc.c)

Message-id: <dec6772483e1eaf722c61a716141f31f05c44e07.1410510773.git.poros@redhat.com>
Patchwork-id: 96002
O-Subject: [PATCH RHEL7.1 182/233] kabi: Adding symbol scsi_is_fc_rport (drivers/scsi/scsi_transport_fc.c)
Bugzilla: 1062361
RH-Acked-by: Rob Evers <revers@redhat.com>

Bugzilla: 1062361
    https://bugzilla.redhat.com/show_bug.cgi?id=1062361

    Description:
    ============
    Used by PowerPath indirectly (via starget_to_rport) to set remote port timeout.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/scsi_is_fc_rport
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/scsi_is_fc_rport
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/scsi_is_fc_rport

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_fc_rport b/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_fc_rport
new file mode 100644
index 0000000..ec01bdc
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/scsi_is_fc_rport
@@ -0,0 +1,2 @@
+#1-
+0x219d8aac scsi_is_fc_rport drivers/scsi/scsi_transport_fc EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/scsi_is_fc_rport b/redhat/kabi/kabi-module/kabi_s390x/scsi_is_fc_rport
new file mode 100644
index 0000000..f9324b7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/scsi_is_fc_rport
@@ -0,0 +1,2 @@
+#1-
+0x72cfc3a3 scsi_is_fc_rport drivers/scsi/scsi_transport_fc EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_fc_rport b/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_fc_rport
new file mode 100644
index 0000000..66e4fab
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/scsi_is_fc_rport
@@ -0,0 +1,2 @@
+#1-
+0xae3cab9b scsi_is_fc_rport drivers/scsi/scsi_transport_fc EXPORT_SYMBOL
-- 
1.7.1